code-conventions.md

```diff
@@ -75,16 +116,19 @@ code. Contributors should also see the dedicated
   - Effect: init ->  execute  -> toString
   - Condition: init -> check -> toString
```

Comment:
```suggestion
  - Condition: init -> check -> toString
  - PropertyCondition: (init) -> check -> (getPropertyType) -> getPropertyName
  - Section: init -> walk -> toString
  - Structure: init -> (preLoad) -> load -> (postLoad) -> unload -> (postUnload) -> (getPriority) -> toString
```
Since we're here :)